src/main/java/com/systemmeltdown/robotlib/subsystems/SingleSpeedTalonDriveSub.java

```diff
@@ -0,0 +1,194 @@
+package com.systemmeltdown.robotlib.subsystems;
+
+import java.util.ArrayList;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.StatusFrame;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.systemmeltdown.robotlib.util.PIDValues;
+import com.systemmeltdown.robotlib.util.Utility;
+
+public class SingleSpeedTalonDriveSub extends SkidSteerDriveSubBase {
+    WPI_TalonSRX m_rightMaster;
+    WPI_TalonSRX m_leftMaster;
+    WPI_TalonSRX[] m_rightSlaves;
+    WPI_TalonSRX[] m_leftSlaves;
+    int m_timeout;
+    int m_talonPidPrimary;
+    double m_driveMotorDeadband;
+    double m_driveRampSeconds;
```

Comment:
Is this a for velocity or proportional? Do we need to store this value at all or just keep it in the Talon controller itself?